### PR TITLE
[py-h5py] added version 3.7.0

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

**Review by adamjstewart — DISMISSED** [review]
LGTM, will give the other maintainers a chance to review before merging.

**Comment by takluyver** [comment]
LGTM, thanks

**Comment by alalazo** [comment]
@spackbot run pipeline

**Review by tgamblin — DISMISSED** [review]

**Review by alalazo — APPROVED** [review]

**Pull request merged**
